ArthurAtangana/SYSC3303A_Project
# Bugfix: last floor was not created, handling bad input

## What type of PR is this? (check all applicable)


## Description

when creating the floors, we had a discrepancy in the for loop that didn't account for indexing at 0 and having x number of floors. we would end up with one less, and the elevator would never unload or load passengers on that floor.

Also, the parser is now not passing bad inputs to the destination dispatcher. This means that if there is an input from the top floor that wants to go up, or an input from the bottom floor going down, it will not even show in the system and be ignored instantly

## How to test

run the scenario 7 and 8 recently added and see if there is a load/unload on the top floor.

run scenario 9 and 10 and see if the first inputs are skipped.

## Added/updated tests?


## Screenshots
